enterprise/server/auth/auth.go

```diff
@@ -833,11 +838,11 @@ func (a *OpenIDAuthenticator) authenticateUser(w http.ResponseWriter, r *http.Re
 	// If the session is not found, bail.
 	sesh, err := authDB.ReadSession(ctx, sessionID)
 	if err != nil {
-		return nil, ut, status.PermissionDeniedError(loggedOutMsg)
+		return nil, ut, status.PermissionDeniedErrorf("%s: session not found: %s", loggedOutMsg, err.Error())
```

Comment:
I'm a little wary about returning the raw error message in auth to the user as it's hard to predict what information we are exposing to a potential attacker.

Reply:
Good point, I've switched to returning a static error and debug logging the actual error.

This gives us the best of both worlds in that we know which error is being hit, and debug logging can be enabled to see the error message.